src/analytics/Suggestions.js

```diff
@@ -54,31 +56,136 @@ Suggestion.propTypes = {
   index: PropTypes.number.isRequired,
 };
 
-const Suggestions = ({ adds, cuts, cube, maybeboard, filter }) => {
+const ImageSuggestion = ({ card, cube }) => {
+  return (
+    <AddModal
+      card={card}
+      href={`/tool/card/${encodeName(card.cardID)}`}
+      modalProps={{ card: card.details, hideAnalytics: false, cubeContext: cube.id }}
+    >
+      <img
+        className="pr-1 w-100"
+        src={card.details.image_normal}
+      />
+    </AddModal>
+  );
+};
+
+ImageSuggestion.propTypes = {
+  card: PropTypes.shape({
+    cardID: PropTypes.string.isRequired,
+    details: PropTypes.shape({
+      image_normal: PropTypes.string.isRequired,
+      image_flip: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+    }).isRequired,
+  }).isRequired,
+  cube: CubePropType.isRequired,
+  index: PropTypes.number.isRequired,
+};
+
+const PAGE_SIZE = 100;
+
+const Suggestions = () => {
+  const { cube, filterInput } = useContext(CubeContext);
   const [maybeOnly, toggleMaybeOnly] = useToggle(false);
+  const [useImages, toggleUseImages] = useToggle(false);
+
+  const [addCards, setAddCards] = React.useState([]);
```

Comment:
It's a little strange to see `useState` accessed through the React namespace whene every other hook is imported explicitly.